setup.py

```diff
@@ -42,6 +42,7 @@
         "ipython>=6.2.1,<7.0.0",
         "plyvel==1.0.5",
         "web3==4.4.1",
+        "lahja==0.4.0",
```

Comment:
I believe that you need to cut another release to remove `aioprocessing` from the `lahja` dependencies as I think it is no longer needed.

Reply:
Ah! Probably! How do you feel about moving that under the ethereum umbrella?

Reply:
Ah! Probably! How do you feel about moving that under the ethereum umbrella?

Reply:
Can you coordinate with devops team.  I think someone will have to click "accept" to migrate the repository.  Have them put it under the "Python-Admin` group for administration.